.circleci/config.yml

```diff
@@ -414,7 +414,8 @@ jobs:
 
   integration-tests:
     executor: in-container
-    resource_class: xlarge
+    resource_class: large
+    parallelism: 2
```

Comment:
This won't work as tests are currently split in the integration tests config. ./circleci/config.yml (integration-tests step) needs to be modifier - the appropriate code is likely in the git history.